packages/mobile/src/account/LinkBankAccountScreen.tsx

```diff
@@ -78,14 +95,32 @@ function StepOne({ kycStatus }: Props) {
         </View>
       )
     default:
-      /* For the Created and Expired state also show the default Begin Verification view */
+      /* Show a spinner while Persona is in progress and we are waiting for IHL to update kycStatus */
+      if (isKycVerifying) {
+        return (
+          <View style={styles.stepOne}>
+            <View style={styles.loadingSpinnerContainer}>
+              <LoadingSpinner width={30} />
+            </View>
+            <Text style={styles.action}>{t('linkBankAccountScreen.verifying.title')}</Text>
+            <Text style={styles.description}>
+              {t('linkBankAccountScreen.verifying.description')}
+            </Text>
+          </View>
+        )
+      }
       return (
         <View style={styles.stepOne}>
           <Text style={styles.label}>{t('linkBankAccountScreen.begin.label')}</Text>
           <Text style={styles.action}>{t('linkBankAccountScreen.begin.title')}</Text>
           <Text style={styles.description}>{t('linkBankAccountScreen.begin.description')}</Text>
           <View style={styles.button}>
-            <PersonaButton kycStatus={kycStatus} text={t('linkBankAccountScreen.begin.cta')} />
+            <PersonaButton
+              kycStatus={kycStatus}
+              text={t('linkBankAccountScreen.begin.cta')}
+              onPress={onPressPersona}
+              onCancelled={() => setIsKycVerifying(false)}
```

Comment:
given that these are always the same, do they need to be props?

Reply:
Yes, because the `setIsKycVerifying` needs to be passed to the persona component somehow.